File: C/homework/6/6.3/3.cpp
#include <stdio.h>
#include <string.h>

int main() {
  // input:
  char str[20];
  printf("enter text: ");
  scanf("%s", str);
  int max = strlen(str), pos;
  printf("enter position to delete char(1-%i): ", max);
  scanf("%i", &pos);
  if (--pos < 0 || pos > max - 1) {
    printf("Error!");
    return 0;
  }  
  // output:
  str[pos] = ' ';
  for (int i = pos; i < max - 1; i++) {
    char save = str[i + 1];
    str[i + 1] = str[i];
    str[i] = save;
  }
  printf("result: %s\n", str);
}

File: Python/2.5/3.py
x = int(input('X: '))
a = int(input('Mike: '))
b = int(input('Ivan: '))

def check(x, a, b):
  if a >= x and b < x: return 'Mike'
  if b >= x and a < x: return 'Ivan'
  if a + b >= x and a < x and b < x: return 1
  if a >= x and b >= x: return 2
  return 0

print('Result:', check(x, a, b))

